contents/handbook/growth/sales/yc-onboarding.md

```diff
@@ -4,39 +4,53 @@ sidebar: Handbook
 showTitle: true
 ---
 
-As a Y Combinator company ourselves, we want to support other YC companies that wish to use PostHog. As such, our onboarding process for YC companies is different and more extensive than our general process.
+We want to support other YC companies using PostHog because:
+
+- Many of these companies will quickly grow into [our ICP](https://posthog.com/handbook/strategy/ideal-customer-persona) and we have an opportunity to get in early with them. 
+- A lot of our most helpful and direct product feedback has generally come from other YC companies. 
+- It's nice to give back to the YC community. 
 
 ## The YC deal
 
-Our YC deal provides the following:
+For startups in the W23 batch:
+
+- Get PostHog Cloud free for 12 months with up to 20m events and 50,000 session recordings per month
+- Choose between a free pair of Airpods or a Timbuk2 backpack
+- Book into a free YC group setup session where we will…
+  - Build a dashboard for the batch with your product growth metrics
+  - Pair with you to ensure you pass calls correctly
```

Comment:
In practice, I'm only equipped to do this in a limited number of scenarios. I have a good handle on where to put the calls etc, but I get out of my depth quickly trying to read other people's code and advise on where, exactly, certain calls should go. I've only been awkwardly unable to keep up once that I can remember, so I guess it's fine to keep this. But usually I try to debug these offline, because I can think them through, read documentation, and ask others for help/to handle if I can't figure it out.

Reply:
Ok, let's leave it as is but iterate the process if you think it needs it once we start getting folks in?